### DISCUSSION TITLE
Save all tagged photos to a chosen directory for img2img batch

### DISCUSSION THREAD

**Discussion by bobpuffer1** [zanllp/infinite-image-browsing#245] [Ideas]
1. Tag photos with "like" or something user-defined
2. Do a search on photos based on tag(s)
3. Select all results from search
4. Send to batch in directory
5. Perform img2img batch

This would be tremendous in selecting and upscaling photos

**Comment by zanllp** [comment]
You can drag the target images into the designated folder by selecting multiple image  on the search results page.
<img width="1904" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/c0202b5d-ceff-42e6-b327-f68330c26f33">

**Reply by bobpuffer1** [reply]
Hmmm. I get my search results next to my file browser (Finder), select multiple files, drag but they don't drop into the file browser. I also consistently get Än error has occurred" when I try to open a search result (single pic) in the local file browser.

**Reply by zanllp** [reply]
It looks like there's a bit of an issue, let me handle it.

**Reply by zanllp** [reply]
@bobpuffer1 It has already been fixed. You can update to the latest version and try it out.

**Reply by bobpuffer1** [reply]
Refreshed -- fails to load. Created issue.

**Comment by bobpuffer1** [comment]
This has been resolved
